runtime/task.go

```diff
@@ -54,6 +55,8 @@ type Process interface {
 type Task interface {
 	Process
 
+	// PID of the process
+	PID() uint32
```

Comment:
Why not on the Process interface?

Reply:
There is currently no pid returned on an exec create, as create only creates the process metadata then start exec's the process

Reply:
Ah I see.